index.js

```diff
@@ -255,6 +266,16 @@ instance.prototype.action = function(action) {
 			args.push({ type: 'i', value: '0' });
 			cmd = '/track/' + opt.track + '/recarm';
 			break;
+		
+		case 'custom_action':
+			// Integer & String commandID's are sent differently...
+			if (parseInt() > 0) {
```

Comment:
This will never be true

Reply:
*sigh* no of course not!....
let me see where I have screwed up and left that out

Reply:
Fixed...sorry about that!
(Wow you are sharp!)

Reply:
Done!